.babelrc

```diff
@@ -0,0 +1 @@
+ { "presets": ["react"] }
```

Comment:
This should probably be in your `package.json` instead, avoids needing the separate file.